crates/sui/src/lib.rs

```diff
@@ -5,11 +5,11 @@
 pub mod client_commands;
 pub mod console;
 pub mod fire_drill;
+pub mod genesis_ceremony;
+pub mod genesis_inspector;
```

Comment:
`genesis_ceremony` and `genesis_inspector` don't appear to be new files and don't appear to be used anywhere in this diff. Do we need them?

Reply:
Good question, I do not know honestly.